spec/controllers/application_submissions_spec.rb

```diff
@@ -341,7 +342,7 @@
       let!(:record) { create :application_submission, saved_for_later: false }
       let(:submit) do
         post :toggle_saved_for_later,
-          params: { id: record.id, commit: 'Save for later', application_submission: { mail_to_applicant: true } }
+             params: { id: record.id, commit: 'Save for later', application_submission: { mail_to_applicant: true } }
```

Comment:
Is there a reason for the extra spacing here?

Reply:
yes, rubocop wanted it.